03/app.js

```diff
@@ -43,16 +43,35 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
+    if(typeof this[isbn] === 'undefined' || typeof this[isbn].title[lang] === 'undefined') {
```

Comment:
Może warto też sprawdzić `typeof this[isbn].title === 'undefined'` bo może się okazać, że w naszej strukturze danych brakuje również tej informacji.